packages/tools/src/utilities/math/polyline/planarFreehandROIInternalTypes.ts

```diff
@@ -0,0 +1,30 @@
+import type { Types } from '@cornerstonejs/core';
+import { PlanarFreehandROIAnnotation } from '../../../types/ToolSpecificAnnotationTypes';
+
+type PlanarFreehandROIDrawData = {
```

Comment:
are these files exported to the app? if yes, we need documentations so that it appears correctly at https://www.cornerstonejs.org/api/tools/namespace/Types

Reply:
They are internal types only used by the drawing loop. I was unsure where to put them. Any ideas? I'll write a comment that they are internal for now.